dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.internal.grpc-1.0.feature

```diff
@@ -11,14 +11,9 @@ IBM-API-Package: \
   io.grpc.stub.annotations;  type="internal", \
   io.grpc.internal; type="internal"
 Subsystem-Name: gRPC internal 1.0
--bundles=\
-  io.openliberty.grpc.1.0.internal.common, \
-  io.openliberty.io.grpc.1.0; location:="dev/api/stable/,lib/"; mavenCoordinates="io.grpc:grpc-api:1.38.1"
 -features=com.ibm.websphere.appserver.appmanager-1.0, \
   com.ibm.websphere.appserver.containerServices-1.0, \
-  com.ibm.websphere.appserver.javax.annotation-1.3; ibm.tolerates:="1.2", \
```

Comment:
Should these have moved to the ee8 / ee9 private features?  Isn't anno still needed?

Reply:
Since I'm moving the placement of servlet, I have added them back in in their respective EE features.

Reply:
NO.  The servlet-4.0 still needs to be in the ee8 file and servlet-5.0 still needs to be in the ee9 file.  The point is that we do tolerate it for the entire feature, and the 4.0 in the ee8 and the 5.0 in the ee9 tell the feature manager which one to choose of the eeX features based off of which of the tolerates was chose for the entire feature.

Reply:
The point of this particular review comment was to ask about the anno features.  They are removed completely and not added to the ee8 / ee9 private features.  Shouldn't they be?  There is annotation function for grpc right?  This review comment was not addressed in your updates.

Reply:
I guess I am blind. They were there in the private eeX features.